### DISCUSSION TITLE
Error when running phpcs

### DISCUSSION THREAD

**Discussion by judodan** [squizlabs/PHP_CodeSniffer#3774] [General]
Came in after several months not working with PHP to find my php not working. I updated versions (to PHP 8.2.3) and things seemed OK. So I upgraded my phpcs to 3.7.2 and ran it on a directory like so:

phpcs -s -p -v --ignore=plugins/,vendor/,_gsdata_/,wp-admin/,wp-includes/,*.css,*.js . --standard=PHPCompatibility --report-file=compat.txt

And I am receiving a number of same errors on each file (below). Google search didn't seem to help me much. Any ideas to give a kick in the right direction?

Errors look like this:

FILE: <file location>
---------------------------------------------------------------------------------------------------------------------------------------------
FOUND 1 ERROR AFFECTING 1 LINE
---------------------------------------------------------------------------------------------------------------------------------------------
 1 | ERROR | An error occurred during processing; checking has been aborted. The error message was: trim(): Passing null to parameter #1
   |       | ($string) of type string is deprecated in /Users/izz/Development/PHPCompatibility/PHPCompatibility/Sniff.php on line 131
   |       | (Internal.Exception)
---------------------------------------------------------------------------------------------------------------------------------------------

**Comment by jrfnl** [comment]
Please report issues with the external [PHPCompatibility standard](https://github.com/PHPCompatibility/PHPCompatibility) in the repo for that standard. This is nothing to do with PHPCS itself.

As a side-note: the issue above has already been fixed in the PHPCompatibility repo. That last release (from 2019) is not compatible with PHP 8.0+. A new release is expected soonish. For the time being, either run PHPCompatibility on PHP 7.4 or use the `develop` version if you want to use PHP 8.0+.

**Reply by judodan** [reply]
Thanks for quick response, trying to get this all straightened out (again!).
